packages/react-liveness/src/components/FaceLivenessDetector/utils/platform.ts

```diff
@@ -0,0 +1,7 @@
+import { VERSION } from '../../../version';
+
+const BASE_USER_AGENT = `ui-react-liveness ${VERSION}`;
```

Comment:
Decided not to include `aws-amplify` or extra `/` because this is already appended after the amplify js version

Did not include `@` symbols because they get converted to spaces

Reply:
Can we do `@ui-react-liveness/${VERSION}` like we did for Auth? Assuming this works, this will sort nicely on our metrics.

https://github.com/aws-amplify/amplify-ui/blob/c85b2eeac7b42b3cd5fbd765f0f27e4a0bec1b8b/packages/ui/src/helpers/authenticator/utils.ts#L21

Reply:
yep we can! my main call out is just that @ will become - when appended to the query string

Reply:
Yep, let's just use `/` then 🙏  `ui-react-liveness/1.0.2`